frontend/src/features/apartment/ApartmentCreatePage.tsx

```diff
@@ -167,7 +167,6 @@ const ApartmentCreatePage = () => {
                             inputType="number"
                             label="Kerros"
                             fieldPath="address.floor"
-                            required
```

Comment:
The `required` attributes are not unnecessary?

Reply:
Quoting Saila's comment to HT-33: "Asunnon luominen ei onnistu pelkkiin tähdellä merkittyjen täyttämisellä, myös ensisijainen laina täytyy täyttää. Pakollisiksi kentiksi riittää asunnon luontivaiheessa pelkästään Rakennus, Rappu, asunnon nro, asuntotyyppi."